src/m365/spo/commands/field/field-remove.ts

```diff
@@ -191,7 +196,8 @@ class SpoFieldRemoveCommand extends SpoCommand {
       }
       else {
         try {
-          await removeField(listRestUrl, args.options.id, args.options.title);
+          const columnName: string | undefined = args.options.title ? args.options.title : args.options.internalName;
```

Comment:
The removeField function now has a title parameter, which can contain the title or the internalName, that does not make sense. Let's just forward the entire options object into the removeField function